guide/14-deep-learning/track_objects_using_SiamMask.ipynb

```diff
@@ -100,7 +100,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Previous:</p><p><strong>Note</strong>&nbsp;The following code is an example depicting usage of SiamMask API, one of the ways in which the API can be used.</p><p> </p><p>Suggested:</p><p>After the heading "Track Object...."</p><p> </p><p>The sample code below depicts usage of SiamMask model using ArcGIS API for Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Done</p>